cylc/flow/cfgspec/globalcfg.py

```diff
@@ -148,74 +150,74 @@
-            'execution polling intervals': [VDR.V_INTERVAL_LIST],
-            'scp command': [
-                VDR.V_STRING, 'scp -oBatchMode=yes -oConnectTimeout=10'],
-            'ssh command': [
-                VDR.V_STRING, 'ssh -oBatchMode=yes -oConnectTimeout=10'],
-            'use login shell': [VDR.V_BOOLEAN, True],
-            'cylc executable': [VDR.V_STRING, 'cylc'],
-            'global init-script': [VDR.V_STRING],
-            'copyable environment variables': [VDR.V_STRING_LIST],
-            'retrieve job logs': [VDR.V_BOOLEAN],
-            'retrieve job logs command': [VDR.V_STRING, 'rsync -a'],
-            'retrieve job logs max size': [VDR.V_STRING],
-            'retrieve job logs retry delays': [VDR.V_INTERVAL_LIST],
-            'task event handler retry delays': [VDR.V_INTERVAL_LIST],
-            'tail command template': [
-                VDR.V_STRING, 'tail -n +1 -F %(filename)s'],
-            'batch systems': {
-                '__MANY__': {
-                    'err tailer': [VDR.V_STRING],
-                    'out tailer': [VDR.V_STRING],
-                    'err viewer': [VDR.V_STRING],
-                    'out viewer': [VDR.V_STRING],
-                    'job name length maximum': [VDR.V_INTEGER],
-                    'execution time limit polling intervals': [
-                        VDR.V_INTERVAL_LIST],
-                },
-            },
-        },
-        '__MANY__': {
-            'run directory': [VDR.V_STRING],
-            'work directory': [VDR.V_STRING],
-            'task communication method': [
-                VDR.V_STRING, 'default', 'ssh', 'poll'],
-            'submission polling intervals': [VDR.V_INTERVAL_LIST],
-            'execution polling intervals': [VDR.V_INTERVAL_LIST],
-            'scp command': [VDR.V_STRING],
-            'ssh command': [VDR.V_STRING],
-            'use login shell': [VDR.V_BOOLEAN],
-            'cylc executable': [VDR.V_STRING],
-            'global init-script': [VDR.V_STRING],
-            'copyable environment variables': [VDR.V_STRING_LIST],
-            'retrieve job logs': [VDR.V_BOOLEAN],
-            'retrieve job logs command': [VDR.V_STRING],
-            'retrieve job logs max size': [VDR.V_STRING],
-            'retrieve job logs retry delays': [VDR.V_INTERVAL_LIST],
-            'task event handler retry delays': [VDR.V_INTERVAL_LIST],
-            'tail command template': [VDR.V_STRING],
-            'batch systems': {
-                '__MANY__': {
-                    'err tailer': [VDR.V_STRING],
-                    'out tailer': [VDR.V_STRING],
-                    'out viewer': [VDR.V_STRING],
-                    'err viewer': [VDR.V_STRING],
-                    'job name length maximum': [VDR.V_INTEGER],
-                    'execution time limit polling intervals': [
-                        VDR.V_INTERVAL_LIST],
-                },
-            },
-        },
-    },
+    # # task
```

Comment:
@dpmatthews Surely we want to allow hosts in the suite config, but they can go in the global?